tensorhive/database.py

```diff
@@ -23,10 +24,42 @@ def init_db() -> None:
     from tensorhive.models.user import UserModel
     from tensorhive.models.reservation_event import ReservationEventModel
     from tensorhive.models.auth import RevokedTokenModel
-    
+    from tensorhive.models.role import RoleModel
+    from tensorhive.models.user.UserModel import UserModel
+
     if database_exists(DB.SQLALCHEMY_DATABASE_URI):
+        if not UserModel.find_by_username('admin'):
+            log.info('[•] Admin has not been found.')
+            create_admin()
         log.info('[•] Database found ({path})'.format(path=DB.SQLALCHEMY_DATABASE_URI))
+
     else:
         # Double check via checkfirst=True (does not execute CREATE query on tables which already exist)
         Base.metadata.create_all(bind=engine, checkfirst=True)
         log.info('[✔] Database created ({path})'.format(path=DB.SQLALCHEMY_DATABASE_URI))
+        create_admin()
+
+def create_admin():
+    from tensorhive.models.user.UserModel import UserModel
+    from tensorhive.models.role.RoleModel import RoleModel
+    admin_default_password  = 'tensorhive_admin'
+    new_user = UserModel(
+        username='admin',
+        password=UserModel.generate_hash(admin_default_password)
+    )
+
+    try:
+        new_user.save_to_db()
+        for role_name in ['user', 'admin']:
+            new_role = RoleModel(
+                name=role_name,
+                user_id=new_user.id
+            )
+            try:
+                new_role.save_to_db()
+            except:
+                log.error('[•] Admin is not created due to role exception.')
+    except:
+        log.error('[•] Admin is not created.')
+
+    log.info('[•] Admin created. Login: {name} . Password : {password} .'.format(name=new_user.username, password=admin_default_password))
```

Comment:
1. It mimics exactly what the `CreateUserController.register(...)` does. I think we should refactor this in some way (not necessarily in this particular PR)
2. Nested `try/except` + `save_to_db` does not throw exceptions, need to reduce it to one level, example:
![image](https://user-images.githubusercontent.com/12485656/46280482-4c2da600-c56c-11e8-84a0-cf38b7725cfe.png)


3. Are you sure we want to display the password? Logs never forget!
Default passwords are always bad (with routers especially ;) )
**In the next PR I'll prompt the user (admin) for custom password instead.**

Reply:
3. I'll do it